icd/api/compiler_solution_llpc.cpp

```diff
@@ -705,4 +748,19 @@ VkResult CompilerSolutionLlpc::CreateLlpcCompiler(
     return (llpcResult == Vkgc::Result::Success) ? VK_SUCCESS : VK_ERROR_INITIALIZATION_FAILED;
 }
 
+// =====================================================================================================================
+// Update the cache feedback for a stage.
+void CompilerSolutionLlpc::UpdateStageCreationFeedback(
+    PipelineCreationFeedback* stageFeedback,
+    const Vkgc::PipelineShaderInfo& shader,
+    const Llpc::CacheAccessInfo* stageCacheAccesses, ShaderStage stage)
```

Comment:
Pointers should have p prefix so stageCacheAccesses should be pStageCacheAccesses.  I see a few places like this.

Reply:
done